### PR TITLE
Adding brem_49 and acgaunt - Not fully working or tested yet 

### PR DESCRIPTION
**Pull request by hayesla**
Adding these here just as a PR to get me started - I'll fix, test it and prettify

### CODE REVIEW

File: sunxspex/brem_49.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import numpy as np 
+import acgaunt
+
+def brem_49(kt, E):
```
Comment (DanRyanIrish) [inline sunxspex/brem_49.py]: FYI: https://github.com/DanRyanIrish/sunxspex/blob/master/sunxspex/models.py#L9

**Comment by hayesla** [comment]
Oh yes! This is just a place holder at the moment for myself

**Comment by hayesla** [comment]
I'll fix that over weekend

File: sunxspex/acgaunt.py [hunk 2]
```diff
@@ -0,0 +1,361 @@
+import numpy as np 
+
+def acgaunt(wave, te_6):
```
Comment (DanRyanIrish) [inline sunxspex/acgaunt.py]: Perhaps this should be a class where the data currently hard coded in the function is in a (or a few) files.  On instantiation of the class the files can be read once and then the function can be a class method which has access to the data without having to reread it every time.

**Comment by DanRyanIrish** [comment]
Thanks @hayesla!  Great start!

**Comment by DanRyanIrish** [comment]
Don't let it interfere with your weekend, or higher priorities.

Although it is nice I'm not the only one with a PR!

File: sunxspex/acgaunt.py (hunk 2 above)
Comment (hayesla) [reply sunxspex/acgaunt.py]: Yep _definitely_ agree, at the moment it's literally just a direct copy of IDL --> Python 😋

Comment (DanRyanIrish) [reply sunxspex/acgaunt.py]: Great!  We're on the same page!

**Comment by DanRyanIrish** [comment]
Hi @hayesla.  What's the status of this PR?  Are there plans to work further on it?

**Comment by hayesla** [comment]
Hi @DanRyanIrish  - I'll be back on it today and will update

**Pull request closed without merging**